docs/developer-guide/printing-module.md

```diff
@@ -0,0 +1,114 @@
+# Printing Module
+
+The **printing module** of MapStore is a back-end service **not included  by default in the release package** that allows to create a printable PDF from the current map.
+
+!!! note
+    The **printing module** is required by the **Print plugin** of MapStore, so if you want to have the Print plugin working in you application, you have to include also the printing module in your MapStore installation.
+
+## Including the printing module in MapStore
+
+Because MapStore doesn't include the printing module by default, to use it you need to build rom the source a `MapStore.war` that includes it or add the missing files to an existing `MapStore.war`
```

Comment:
recheck this sentence because it is missing something especially the part

> rom the source a `MapStore.war`

Reply:
An `f` is missing, I think :cry: 
```suggestion
Because MapStore doesn't include the printing module by default, to use it you need to build from the source a `MapStore.war` that includes it OR add the missing files to an existing `MapStore.war`
```